fms/models/gpt_bigcode.py

```diff
@@ -30,6 +32,7 @@ class GPTBigCodeConfig(ModelConfig):
     emb_dropout: float = 0.0
     multiquery_attn: bool = True
     ln_eps: float = 1e-5
+    gptq_config: Optional[GPTQConfig] = None
```

Comment:
rename to `linear_config`, make it Optional[Dict[LinearEnum, Any]] where the LinearEnum is the one with GPTQ, FP8, etc.

Reply:
```
class LinearType(Enum):
    PYTORCH = 1
    GPTQ = 2
    FP8 = 3

class LinearConfig:
    type: LinearType
    config: Any
```